packages/lodestar/src/metrics/metrics/lodestar.ts

```diff
@@ -367,6 +367,20 @@ export function createLodestarMetrics(
       }),
     },
 
+    // BLS time
+    blsTime: {
+      singleThreadDuration: register.histogram({
+        name: "lodestar_bls_single_thread_time_seconds",
+        help: "Time to verify signatures with single thread mode",
+        buckets: [0.1, 1],
+      }),
+      mainThreadDurationInThreadPool: register.histogram({
```

Comment:
There's already a group `blsThreadPool`, add `mainThreadDurationInThreadPool` to there